enterprise_edition/theming.mdx

```diff
@@ -14,6 +14,39 @@ As part of the Enterprise Edition, we provide the ability to easily customize th
 To provide your own application name / logo, you can use the `Whitelabeling` tab in the admin panel. Here, we provide a way to input
```

Comment:
It seems like we use different spellings for white-labeling / whitelabelling across the app. Let's aim to make this consistent.